frontend/src/toolbar/bar/Toolbar.tsx

```diff
@@ -178,19 +184,32 @@ export function Toolbar(): JSX.Element {
                 }
             >
                 <ToolbarButton
-                    icon={<IconLogomark />}
                     onClick={isAuthenticated ? toggleMinimized : authenticate}
                     title={isAuthenticated ? 'Minimize' : 'Authenticate the PostHog Toolbar'}
                     titleMinimized={isAuthenticated ? 'Expand the toolbar' : 'Authenticate the PostHog Toolbar'}
-                />
+                >
+                    <IconLogomark />
+                </ToolbarButton>
                 {isAuthenticated ? (
                     <>
-                        <ToolbarButton icon={<IconSearch />} menuId="inspect" />
-                        <ToolbarButton icon={<IconCursorClick />} menuId="heatmap" />
-                        <ToolbarButton icon={<IconBolt />} menuId="actions" />
-                        <ToolbarButton icon={<IconToggle />} menuId="flags" title="Feature flags" />
+                        <ToolbarButton menuId="inspect">
+                            <IconSearch />
+                        </ToolbarButton>
```

Comment:
is the icon change some react foo i don't understand or just its pleasing to your eye (or something else?) :)

Reply:
It's just there is no difference between the text or an icon - they were just children so I made it standard children